### PR TITLE
docs: fix links in tool_guide.md

### PR DESCRIPTION
**Pull request by lakshyaag**
**Description**: Update the links from `/toolkit` to `/cohere-toolkit`
**Issues**: None
**Dependencies**: None

### CODE REVIEW

File: docs/custom_tool_guides/tool_guide.md [hunk 1]
```diff
@@ -203,4 +203,4 @@ curl --location 'http://localhost:8000/chat-stream' \
 
 ## Step 6 (extra): Add Unit tests
 
-If you would like to go above and beyond, it would be helpful to add some unit tests to ensure that your tool is working as expected. Create a file [here](https://github.com/cohere-ai/toolkit/tree/main/src/backend/tests/tools) and add a few cases.
\ No newline at end of file
+If you would like to go above and beyond, it would be helpful to add some unit tests to ensure that your tool is working as expected. Create a file [here](https://github.com/cohere-ai/toolkit/tree/main/src/backend/tests/tools) and add a few cases.
```
Comment (lusmoura) [inline docs/custom_tool_guides/tool_guide.md:206]: It seems like this one was not updated 😅 Other than that, LGTM!

Comment (lakshyaag) [reply docs/custom_tool_guides/tool_guide.md]: Ah my bad - fixed it!

**Comment by lusmoura** [comment]
Thank you for your contribution! We appreciate it and hope to see more of your contributions in the future 😄
BTW you'll need to update your branch to be able to merge it.

**Review by lusmoura — APPROVED** [review]
Thank you for your contribution! We appreciate it and hope to see more of your contributions in the future 😄

**Comment by tianjing-li** [comment]
@lakshyaag fixed the merge conflicts for you

**Comment by lakshyaag** [comment]
Thank you @lusmoura and @tianjing-li - I think the merge is blocked because of review approval.

**Review by lusmoura — APPROVED** [review]

**Pull request merged**